src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs

```diff
@@ -57,66 +57,55 @@ internal class LocalGrpcListener : IHostedService, IDisposable
 
         public void Dispose()
         {
-            this.host?.Dispose();
+            this.lifetime?.SignalShutdown();
+            this.lifetime?.Dispose();
         }
 
-        public async Task StartAsync(CancellationToken cancelToken = default)
+        public Task StartAsync(CancellationToken cancelToken = default)
```

Comment:
Why did we end up dropping the `async` keyword?

Reply:
I see in the StopAsync method we actually add the `async` keyword, which also throws me off :P . Is this all somehow related to async support in .net6?

Reply:
> Why did we end up dropping the async keyword?

Because we no longer use `await` in this method (there's no async methods being called anymore).

> I see in the StopAsync method we actually add the async keyword, which also throws me off :P . Is this all somehow related to async support in .net6?

No, it's simply an accidental artifact of refactoring the code to use different APIs. StopAsync can be implemented with or without `async`.